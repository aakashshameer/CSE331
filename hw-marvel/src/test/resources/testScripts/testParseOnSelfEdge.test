# test Parser on SelfEdge data to make sure MarvelPath
# doesn't allow building reflexive edges

# Parse through the data and build the graph
LoadGraph selfEdge selfEdge.csv

# List nodes and edges to see if the graph is built correctly
ListNodes selfEdge
ListChildren selfEdge A
ListChildren selfEdge B
ListChildren selfEdge C