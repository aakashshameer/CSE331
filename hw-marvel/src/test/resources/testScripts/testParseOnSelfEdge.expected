# test Parser on SelfEdge data to make sure MarvelPath
# doesn't allow building reflexive edges

# Parse through the data and build the graph
loaded graph selfEdge

# List nodes and edges to see if the graph is built correctly
selfEdge contains: A B C
the children of A in selfEdge are: B(AB)
the children of B in selfEdge are: A(AB) C(BC)
the children of C in selfEdge are: B(BC)